sdk/webpubsub/Azure.Messaging.WebPubSub.Client/src/Internals/MemoryBufferWriter.cs

```diff
@@ -197,11 +197,18 @@ private async Task CopyToSlowAsync(Stream destination)
                 for (var i = 0; i < count; i++)
                 {
                     var segment = _completedSegments[i];
+#if NETSTANDARD2_1_OR_GREATER
```

Comment:
Your library is no longer target .NET STANDARD 2.1 or higher. RequiredTargetFrameworks is set to netstandard2.0 only now with your change to the project.